### ISSUE TITLE
I get the following error when trying to run it. TypeError: text is not iterable

### ISSUE THREAD

**Issue by LordVictor0424** [theskollie/truth-puppeteer#5]
C:\Users\rbt04\truth-puppeteer>npm run truthbot
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> truth-social-bot@1.0.0 truthbot
> node dist/index.js

[16:48]  Script Starting! No further inputs required.
C:\Users\rbt04\truth-puppeteer\node_modules\puppeteer\lib\cjs\puppeteer\common\Input.js:222
        for (const char of text) {
                           ^

TypeError: text is not iterable
    at Keyboard.type (C:\Users\rbt04\truth-puppeteer\node_modules\puppeteer\lib\cjs\puppeteer\common\Input.js:222:28)
    at ElementHandle.type (C:\Users\rbt04\truth-puppeteer\node_modules\puppeteer\lib\cjs\puppeteer\common\JSHandle.js:749:35)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async DOMWorld.type (C:\Users\rbt04\truth-puppeteer\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:318:9)
    at async login (C:\Users\rbt04\truth-puppeteer\dist\login.js:9:5)
    at async main (C:\Users\rbt04\truth-puppeteer\dist\index.js:27:5)

C:\Users\rbt04\truth-puppeteer>

**Comment by theskollie** [comment]
Seems your login is failing because:
1) You didn't setup the .env file in the root directory, not in the dist file - Check the docs on the repo
2) Your TRUTHSOCIAL is a URL. It must just be your username on Truth

Check this and let me know.

Also, I just pushed a fix for the initial check - So pull the new commit - But this is not a fix for your issue.

**Comment by LordVictor0424** [comment]
The .env file is in the folder 'truth-puppeteer' not in 'dist' folder. The name is 'example.env', should it have a different name?
I checked the .env file and the username is my username on truthsocial, should I include the '@'?

**Comment by theskollie** [comment]
Yes, the file name should be .env - So remove example from the name.
No @ in the Username.
Also, don't browse Truth while the script is running or you will get rate limited by Truth - I'll add this to the docs.
Send me a message on Discord if you don't get it solved : Skollie#9829
